modules/persistent-storage-csi-ebs-operator-install.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+
+[id="persistent-storage-csi-ebs-operator-install_{context}"]
+= Installing the AWS Elastic Block Store CSI Operator
+
+The AWS Elastic Block Store (EBS) Container Storage Interface (CSI) Operator enables the replacement of the existing AWS EBS in-tree storage plug-in.
+
+When you install the AWS EBS CSI Operator, {product-title} provides the CSI driver that allows you to use CSI volumes with the `PersistentVolumeClaims`, `PersistentVolumes`, and `StorageClasses` API objects.
```

Comment:
nit: I'm not sure if it's accurate to say the _OCP will provide the CSI driver_ because it's the operator who installs the driver.

I'm not certain about the wording, so feel free to ignore this comment if I'm mistaken.

Reply:
Same concern here. Do we need to describe how to create csi driver in this doc?

Reply:
Thanks! I updated this for accuracy: "Installing the AWS EBS CSI Operator provides the CSI driver that allows you to use CSI volumes with the `PersistentVolumeClaims`, `PersistentVolumes`, and `StorageClasses` API objects in OCP."
We suggest to users that they refer to third-party docs for instructions on CSI drivers, so we shouldn't describe how to create the driver here.